tfhe/src/integer/client_key/radix.rs

```diff
@@ -27,8 +27,7 @@ use serde::{Deserialize, Serialize};
 ///
 /// let ct = cks.encrypt(msg);
 ///
-/// // Decryption
-/// let dec = cks.decrypt(&ct);
+/// /// let dec = cks.decrypt(&ct);
```

Comment:
this looks wrong

Reply:
I think it's clearer to have the mod only containing other modules (and possibly reexports)